installation/resources/crds/application-connector/applications.applicationconnector.crd.yaml

```diff
@@ -43,6 +43,9 @@ spec:
                   type: boolean
                 skipVerify:
                   type: boolean
+                encodeUrl:
+                  type: boolean
+                  default: true
```

Comment:
Did you test the behavior when you apply a CR without this field? And some upgrade scenarios where there are already CRs present in the cluster and you upgrade the CRD?

Reply:
I did try upgrading from `main` to my PR, with `Application` present and it does get updated with default value for this field (although after some research I don't know if this is the cleanest way to go about it? Should I bump the version?).